package/gluon-config-mode-domain-select/check_site.lua

```diff
@@ -1 +1,2 @@
 need_boolean(in_domain({'hide_domain'}), false)
+need_array_of(in_domain({'hide_domain_codes'}), table_keys(need_table(in_domain({'domain_names'}), nil, true)), false)
```

Comment:
This would be more readable if you assigned the return value of `table_keys(need_table(in_domain({'domain_names'})))` to a local variable. A value is required by default, so the `, nil, true` can be omitted.